e2e-flap.sh

```
@@ -0,0 +1,789 @@
+    local svc="$1" vip port kind
+    case "$svc" in
+        "$SVC_A")      vip="$VIP_A";      port=80;  kind=http ;;
+        "$SVC_MANUAL") vip="$VIP_MANUAL"; port=80;  kind=http ;;
+        "$SVC_B")      vip="$VIP_B";      port=443; kind=tcp  ;;
+        *) return 1 ;;
+    esac
+    [ -z "$vip" ] && return 1
+    # Two attempts: a single wget/nc failure is noisy enough to have produced a
+    # false "offline" reading in an earlier run of this harness.
+    local attempt
+    for attempt in 1 2; do
```

Shellcheck (0.11.0)

[warning] attempt appears unused. Verify use (or export if used externally).

(SC2034)